resources/mixins/kubernetes/jsonnetfile.lock.json

```diff
@@ -18,7 +18,7 @@
           "subdir": "grafana-builder"
         }
       },
-      "version": "dac760854501e85dd2761160249f08aafff99a85",
+      "version": "c0d0a133a88a1a1b0729525d0269e861a28ce09a",
```

Comment:
Why did you update the mixins?

Reply:
It does that automatically when i run `make generate` ..